openlibrary/plugins/openlibrary/js/readinglog_stats.js

```diff
@@ -144,8 +144,8 @@ export function init(config) {
             WHERE {
               VALUES (?x ?olid) {
                 ${authors.filter(a => a.remote_ids.wikidata).map(
-                  a => `"(wd:${a.remote_ids.wikidata} ${a.key.split('/')[2]})"`
-                ).join(' ')}
+        a => `"(wd:${a.remote_ids.wikidata} ${a.key.split('/')[2]})"`
+    ).join(' ')}
```

Comment:
… but why? idk what linter/setting results in this, but i really fail to see how this is an improvement to the code in any way.